CHANGELOG.rst

```diff
@@ -10,6 +10,9 @@ This document describes changes between each past release.
 **New features**
 
 - Include Python 3.7 support.
+- kinto.core.events now supports ``resource_name`` and
```

Comment:
`kinto.core.events.notify_resource_event()` now supports...

Reply:
Should it be added to http://kinto.readthedocs.io/en/stable/core/index.html ?

Reply:
I added it using ``autofunction`` to ``notifications.rst``. On the one hand, you could say I was trying to integrate my documentation with what was already there. On the other hand, you could say I was being lazy...